src/lib/utils/linking.web.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import { Linking } from 'react-native-web'
+
+const schemeRe = /(.+?:)\/\//
+
+const openLink = async (uri: string, target?: '_blank' | '_self' = '_blank', noopener?: boolean = false) => {
```

Comment:
this method isnt used anywhere

Reply:
@sirpy as we had changes in utils/linking, i've asked @nicastelo  to merge with our latest changes and create .native & .web implementation of Linking utiliy

openLink is used on Phase 0 version of the Claim page , on the master branch

Reply:
@nicastelo as your linking utility is extension of RN's Linking, better to do on the web version this way.
So, let linking.web will export RN's Linking copy with openLink overridden

Reply:
e.g 

```javascript
import { Linking } from 'react-native-web'

const schemeRe = /(.+?:)\/\//

const openLink = async (uri: string, target?: '_blank' | '_self' = '_blank', noopener?: boolean = false) // rename fn
// fn code

export default { ...Linking, openURL }

// usage:

// was before:
import openUrl from 'utils/linking'
openLink("http://google.com", "_blank")

// now
import Linking from 'utils/linking'
Linking.openURL("http://google.com", "_blank")
```

Reply:
@serdiukov-o-nordwhale We’ve been going through the comments and I think there's a confusion about Linking and DeepLinking, they’re different things. The previous Linking class created on this PR solves the problem to external links for mobile (saves inside Async Storage a pathname and params to afterwards redirect to this saved route). What we understand from your suggestion is that linking solves the problem to open a URL in a new tab or similar things in web.
What we suggest is to rename the files to DeepLinking and keep using the old Class whose objective is to get a pathname and params, as it suits our needs in mobile and they’re different of what you currently have for web linking.

Reply:
Okay so, pls rename your class to the DeepLinking and you could remove our Linking as it's not used on your branch